README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
We won't double search, I have a check in place that if we are already on the search results for that search text `performSearch` will immediately resolve it's promise.

See here:  https://github.com/gmusic-utils/gmusic-ui.js/pull/3/files#diff-746e6474fa5a0f2ddc268f7e7b33cd15R157